src/services/state-persistor/utils/GeometryTypes.ts

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
I am not sure this file should be in `src/services/state-persistor/`

Reply:
moved to inline type definition